README.md

```diff
@@ -10,6 +14,7 @@
  * Here is an example workflow to [convert FASTQ files to uBAM files](docs/example-fastq-to-ubam.md/#Example-workflow-to-convert-FASTQ-files-to-uBAM-files)<br/>
  * Have an existing WDL file that you want to run on Azure? [Modify your existing WDL with these adaptations for Azure](docs/change-existing-WDL-for-Azure.md/#How-to-modify-an-existing-WDL-file-to-run-on-Cromwell-on-Azure)<br/>
  * Want to run commonly used workflows? [Find links to ready-to-use workflows here](#Run-Common-Workflows)<br/>
+ * Are there any examples of tertiary analysis or other genomics analysis? [Find links to related project here](#Related-Projects)<br/>
 
 #### Questions?

```

Comment:
The second bullet, "Known issues and work-arounds . . ." needs a period at the end of the sentence, or the other bullet needs the period removed. Looks like there are some inconsistencies with this overall.